src/middleware/packages/auth/mixins/auth.js

```diff
@@ -73,33 +122,28 @@ const AuthMixin = {
       }
 
       if (method === 'Bearer') {
-        const payload = await ctx.call('auth.jwt.verifyToken', { token });
+        const payload = await ctx.call('auth.jwt.verifyServerSignedToken', { token });
         if (payload) {
           ctx.meta.tokenPayload = payload;
           ctx.meta.webId = payload.webId;
           return Promise.resolve(payload);
         }
+
+        // Check if token is a capability.
+        if (route.opts.authorizeWithCapability) {
+          return this.validateCapability(ctx, token);
+        }
```

Comment:
Is it really necessary to indicate which routes accept capabilities ? I'm afraid this is something we will forget. Also it's not standard in Moleculer ApiService so it's hard to document.